servicex/templates/did-finder-cernopendata/deployment.yaml

```diff
@@ -21,6 +21,10 @@ spec:
         env:
           - name: INSTANCE_NAME
             value: {{ .Release.Name }}
+        {{- if .Values.didFinder.CERNOpenData.cachePrefix }} 
```

Comment:
Is there any support for this in the CERNOpenData DID Finder? If not let's not confuse people by adding options that don't do anything. We can add this when the DID finder supports it

Reply:
@ivukotic  indicated that the CernOpenData finder supports this, I'm planning on adding support there in consultation with @gordonwatts once the rucio finder support has landed (Ilija and I are testing that right now)